src/EventMeta.php

```diff
@@ -654,7 +654,7 @@ public function addDefaultAccess() {
    *   Whether an entity type uses a separate bundle entity type.
    */
   protected function entityTypeHasBundles($entity_type_id) {
-    $entity_type = $this->entityManager->getDefinition($entity_type_id);
+    $entity_type = $this->entityTypeManager->getDefinition($entity_type_id);
```

Comment:
Entity manager addressed in `d9-fixes` branch where I did split commits on per d-org issue

Reply:
Basically rename of the variable is useless at the moment